bool gesture_found
int32 predicted_class
int32 expected_class
